assets/scss/utilities/_ellipsis.scss

```diff
@@ -0,0 +1,29 @@
+////
+/// Ellipsis
+/// @group utilities
+/// @access private
+/// @author Damián Muti
+////
+
+/// Create ellipsis on an element.
```

Comment:
Is this the form of documentation you were suggesting @damianmuti ?

Reply:
Exactly, @cbernat . It makes use of SassDoc comment structure to parse it and create a separated technical documentation based on it.